Readme.md

```diff
@@ -1,8 +1,28 @@
-install
+Install
 -------
+To install a running SurfCaptain Instance you should use the distribution Package that holds all the
+composer dependencies. Follow theses steps to enter the Setup:
 
 	git clone git@github.com:lightwerk/SurfCaptainDistribution.git
 	cd surfcaptaindistribution
 	composer install
+	copy the content of Packages/Application/Lightwerk.SurfCaptain/Configuration/Routes.yaml to Configuration/Routes.yaml
```

Comment:
Just saw 2 fresh installation on T3DD where this had to be done manually. Didnt digg into it but does not work magically out of the box.